src/components/GatedResourceLayout/index.tsx

```diff
@@ -132,20 +132,43 @@ export const GatedResourceLayout: FunctionComponent<Props> = ({
                 </div>
             </section>
 
-            <section className="bg-white py-6 pb-md-8">
-                <ContentSection className="d-flex flex-column-reverse flex-md-row">
-                    {description}
-
-                    <div className="col-md-6 col-12 pb-md-0 pb-6 px-0">
-                        <h2 className="font-weight-bold">{formLabel}</h2>
-                        <div className="border-saturn border border-3 shadow-sm py-4 px-4 mt-3 px-0">
-                            <div id="form" />
+            {/* ---- DEFAULT BODY VARIATION ---- */}
+            {description && (
+                <section className="bg-white py-6 pb-md-8">
+                    <ContentSection className="d-flex flex-column-reverse flex-md-row">
+                        {description}
+
+                        <div className="col-md-6 col-12 pb-md-0 pb-6 px-0">
+                            <h2 className="font-weight-bold">{formLabel}</h2>
+                            <div className="border-saturn border border-3 shadow-sm py-4 px-4 mt-3 px-0">
+                                <div id="form" />
+                            </div>
                         </div>
-                    </div>
-                </ContentSection>
+                    </ContentSection>
 
-                {children}
-            </section>
+                    {children}
+                </section>
+            )}
+
+            {/* ---- RECORDING BODY VARIATION ---- */}
+            {videoSrc && learnMoreCTA && (
```

Comment:
I think `videoSrc` would be sufficient and you don't necessarily need both conditions. But, with my recent suggestion of using a query param, you can use that to conditionally show this content.